sangwoo/addMeals_problemSolving.java

```diff
@@ -78,4 +72,24 @@ else if(itemCnt[k-1] != itemCnt[k]){
     
     
     
-    }
\ No newline at end of file
+    }
+
+//구독주문상품조회 mealList
+//구독주문마켓상품조회, 구독주문식단상품조회 mealItemList 뭔차이?
+//qty는 배송되어야하는 상품 개수, mealItemList.size()는 상품이 몇 종류인지
+//mealItemList.size()는 qty보다 클일이 거의 없다
+private int[] setMealItemTypeIdx(int qty, int mealItemListSize){
+    int[] mealItemTypeIdx = new int[qty];
+    for(int i=0; i<qty; i++) mealItemTypeIdx[i] = i;
```

Comment:
https://stackoverflow.com/questions/3790142/java-equivalent-of-pythons-rangeint-int

```suggestion
    int[] mealItemTypeIdx = IntStream.range(0, qty)
```